### PR TITLE
docs: add @policy directive

### PR DESCRIPTION
**Pull request by shorgi**
Document `@policy` as a federation directive. [Deploy preview of new @policy section](https://deploy-preview-2867--apollo-federation-docs.netlify.app/federated-types/federated-directives#policy).

### CODE REVIEW

**Review by dariuszkuc — CHANGES_REQUESTED** [review]
Please also update following with new `@policy` directive info
* https://www.apollographql.com/docs/federation/federation-versions
* https://www.apollographql.com/docs/federation/subgraph-spec#subgraph-schema-additions

File: docs/source/federated-types/federated-directives.mdx [hunk 1]
```diff
@@ -417,6 +417,54 @@ Indicates to composition that the target element is accessible only to the authe
 </tbody>
 </table>
 
+### `@policy`
+
+> ⚠️ **This directive is available in Apollo Federation 2.6 and later.**
+> **Is is an [Enterprise feature](/router/enterprise-features) of the Apollo Router** and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).<br/>
```
Comment (chandrikas) [inline docs/source/federated-types/federated-directives.mdx:423]:
```suggestion
> **It is an [Enterprise feature](/router/enterprise-features) of the Apollo Router** and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).<br/>
```

Comment (chandrikas) [reply docs/source/federated-types/federated-directives.mdx:423]: This typo exists in the @requiresScopes block too

File: docs/source/federation-versions.mdx [hunk 2]
```diff
@@ -21,6 +21,103 @@ This article describes notable changes and additions introduced in each minor ve
...
+
+</div>
+
+<div>
+
+Minimum router version
+
+**`1.35.0`**
+
+</div>
```
Comment (shorgi) [inline docs/source/federation-versions.mdx:52]: @Geal - what should this `Minimum router version` be for `@policy` support with Federation 2.6.0?

Comment (dariuszkuc) [reply docs/source/federation-versions.mdx:52]: Compatible router wasn't released yet (and this feature is not yet available in GraphOS). We can update this once router is released and it is available in studio.

Comment (shorgi) [reply docs/source/federation-versions.mdx:52]: @dariuszkuc thanks, changing for now to unavailable in GraphOS and min router version of 1.34.1

Comment (dariuszkuc) [reply docs/source/federation-versions.mdx]: 1.34.1 is not compatible, 1.35.0 (unreleased) might be

Comment (dariuszkuc) [reply docs/source/federation-versions.mdx]: 1.34.1 is not compatible, 1.35.0 might be but is still unreleased. IMHO we should just put TBD and fix it once we got router release

Comment (shorgi) [reply docs/source/federation-versions.mdx]: @dariuszkuc as suggested, setting to TBD for now

**Review by dariuszkuc — APPROVED** [review]

**Review by chandrikas — APPROVED** [review]

**Pull request merged**